static/src/javascripts/projects/journalism/views/politicsWeeklyTreat.html

```diff
@@ -0,0 +1,37 @@
+                <svg class="politics-weekly-treat__player-wave politics-weekly-treat__player-wave-large" width="137px" height="24px" viewBox="0 0 137 24" xmlns="http://www.w3.org/2000/svg"><title>Group</title><g id="Complete" fill="none" fill-rule="evenodd"><g id="5.0_Treat-Politics-NoDuration-Podcast" transform="translate(-71 -765)" fill="#999999"><g id="Group" transform="translate(71 765)"><path id="Rectangle-2" d="M3 9H5V14H3zM54 9H56V14H54zM63 9H65V14H63zM81 9H83V14H81zM96 9H98V14H96zM90 9H92V14H90zM105 9H107V14H105zM111 9H113V14H111zM114 9H116V14H114zM129 9H131V14H129zM18 10H20V14H18zM0 10H2V13H0zM33 7H35V16H33zM42 9H44V14H42zM48 7H50V16H48zM69 7H71V16H69zM123 7H125V16H123zM78 8H80V15H78zM126 8H128V15H126zM57 7H59V16H57zM45 4H47V19H45zM72 4H74V19H72zM120 4H122V19H120zM60 4H62V19H60zM6 5H8V19H6zM12 5H14V19H12zM9 2H11V22H9zM24 3H26V21H24zM36 2H38V22H36zM27 0H29V24H27zM21 5H23V19H21zM30 5H32V19H30zM39 5H41V18H39zM75 6H77V17H75zM117 6H119V17H117zM108 6H110V17H108zM87 5H89V18H87zM102 5H104V18H102zM51 10H53V13H51zM66 10H68V13H66zM84 10H86V13H84zM93 10H95V13H93zM99 10H101V13H99zM132 10H134V13H132zM135 10H137V13H135zM15 8H17V16H15z"/></g></g></g></svg>
+                <svg class="politics-weekly-treat__player-wave politics-weekly-treat__player-wave-small" width="107px" height="24px" viewBox="0 0 107 24" xmlns="http://www.w3.org/2000/svg"><title>Group</title><g id="Complete" fill="none" fill-rule="evenodd"><g id="2.0_Treat-Politics-NoDuration-Podcast" transform="translate(-96 -765)" fill="#999999"><g id="Group" transform="translate(96 765)"><path id="Rectangle-2" d="M3 9H5V14H3zM54 9H56V14H54zM63 9H65V14H63zM81 9H83V14H81zM96 9H98V14H96zM90 9H92V14H90zM105 9H107V14H105zM18 10H20V14H18zM0 10H2V13H0zM33 7H35V16H33zM42 9H44V14H42zM48 7H50V16H48zM69 7H71V16H69zM78 8H80V15H78zM57 7H59V16H57zM45 4H47V19H45zM72 4H74V19H72zM60 4H62V19H60zM6 5H8V19H6zM12 5H14V19H12zM9 2H11V22H9zM24 3H26V21H24zM36 2H38V22H36zM27 0H29V24H27zM21 5H23V19H21zM30 5H32V19H30zM39 5H41V18H39zM75 6H77V17H75zM87 5H89V18H87zM102 5H104V18H102zM51 10H53V13H51zM66 10H68V13H66zM84 10H86V13H84zM93 10H95V13H93zM99 10H101V13H99zM15 8H17V16H15z"/></g></g></g></svg>
+            </div>
```

Comment:
Can we keep the SVGs in separate `.svg` files, import them into the JavaScript and then inject them into this HTML template?

Reply:
The downside to this is that the classes will have to be added in the separate svg files, unless there's a way to add classes when I reference the passed in svgs?

Reply:
Can we wrap each `<svg>` in a `<div>` with the class?

```html
<div class="politics-weekly-treat__player-wave politics-weekly-treat__player-wave-small">
  <svg ...>
</div>
```

```css
.politics-weekly-treat__player-wave > svg {

}

.politics-weekly-treat__player-wave-small > svg {

}
```

Reply:
Isn't that what the `svg` module is supposed to do?
```js
import icon from '.../icon.svg';
const html = svg(icon, ["politics-weekly-treat__player-wave"]);
```

Reply:
@regiskuckaertz very useful, thanks

Reply:
Ah yes, nice one @regiskuckaertz. Although before you use his, it's possible [`svg-loader`](https://github.com/guardian/frontend/blob/master/dev/webpack-loaders/svg-loader/index.js) is doing a similar thing, giving you a class to hook into based on the filename of the SVG. Might be worth a look